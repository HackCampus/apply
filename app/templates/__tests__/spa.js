const test = require('ava')

const shell = require('../shell')

test('shell', t => {
  t.true(typeof shell === 'function')
  t.snapshot(shell('foo'))
  t.snapshot(shell('bar'))
})
