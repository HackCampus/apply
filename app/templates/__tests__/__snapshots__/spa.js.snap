exports[`shell 1`] = `"\"\\n<!DOCTYPE html>\\n<html>\\n<head>\\n<meta charset=\\\"utf-8\\\">\\n<meta property=\\\"og:url\\\" content=\\\"https://hackcampus-apply.herokuapp.com\\\" />\\n<meta property=\\\"og:title\\\" content=\\\"HackCampus\\\" />\\n<meta property=\\\"og:description\\\" content=\\\"We help the brightest students find software engineering internships at London\'s best startups. Apply now!\\\" />\\n<meta property=\\\"description\\\" content=\\\"We help the brightest students find software engineering internships at London\'s best startups. Apply now!\\\" />\\n<meta property=\\\"og:image\\\" content=\\\"https://hackcampus.github.io/images/cover-light.png\\\" />\\n<meta property=\\\"og:image:type\\\" content=\\\"image/png\\\" />\\n<meta property=\\\"og:image:width\\\" content=\\\"1656\\\" />\\n<meta property=\\\"og:image:height\\\" content=\\\"628\\\" />\\n<title>HackCampus</title>\\n<link rel=\\\"stylesheet\\\" href=\\\"/static/common.css\\\" />\\n<link rel=\\\"stylesheet\\\" href=\\\"/static/foo/styles.css\\\" />\\n<link href=\\\"https://fonts.googleapis.com/css?family=Roboto+Mono|Roboto\\\" rel=\\\"stylesheet\\\">\\n</head>\\n<body>\\n<div id=\\\"container\\\"></div>\\n<script src=\\\"https://cdn.polyfill.io/v2/polyfill.min.js\\\"></script>\\n<script src=\\\"/static/foo/index.js\\\"></script>\\n<script>\\n(function(s,p,y,w,a,r,e){s[\'GoogleAnalyticsObject\']=a;s[a]=s[a]||function(){\\n(s[a].q=s[a].q||[]).push(arguments)},s[a].l=1*new Date();r=p.createElement(y),\\ne=p.getElementsByTagName(y)[0];r.async=1;r.src=w;e.parentNode.insertBefore(r,e)\\n})(window,document,\'script\',\'//www.google-analytics.com/analytics.js\',\'ga\');\\nga(\'create\', \'UA-88923602-2\', \'auto\');\\nga(\'send\', \'pageview\');\\n</script>\\n</body>\\n</html>\\n\""`;

exports[`shell 2`] = `""
<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<meta property="og:url" content="https://hackcampus-apply.herokuapp.com" />
<meta property="og:title" content="HackCampus" />
<meta property="og:description" content="We help the brightest students find software engineering internships at London's best startups. Apply now!" />
<meta property="description" content="We help the brightest students find software engineering internships at London's best startups. Apply now!" />
<meta property="og:image" content="https://hackcampus.github.io/images/cover-light.png" />
<meta property="og:image:type" content="image/png" />
<meta property="og:image:width" content="1656" />
<meta property="og:image:height" content="628" />
<title>HackCampus</title>
<link rel="stylesheet" href="/static/common.css" />
<link rel="stylesheet" href="/static/bar/styles.css" />
<link href="https://fonts.googleapis.com/css?family=Roboto+Mono|Roboto" rel="stylesheet">
</head>
<body>
<div id="container"></div>
<script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
<script src="/static/bar/index.js"></script>
<script>
(function(s,p,y,w,a,r,e){s['GoogleAnalyticsObject']=a;s[a]=s[a]||function(){
(s[a].q=s[a].q||[]).push(arguments)},s[a].l=1*new Date();r=p.createElement(y),
e=p.getElementsByTagName(y)[0];r.async=1;r.src=w;e.parentNode.insertBefore(r,e)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
ga('create', 'UA-88923602-2', 'auto');
ga('send', 'pageview');
</script>
</body>
</html>
""`;
